lib/fastlane_core/itunes_connect/itunes_connect_helper.rb

```diff
@@ -61,8 +61,9 @@ def wait_for_preprocessing
         end
       end
 
-      def wait_for_elements(name)
+      def wait_for_elements(name, include_hidden = false)
         counter = 0
+        Capybara.ignore_hidden_elements = false if include_hidden
```

Comment:
I just noticed, this is behaving incorrectly. Assume you set `ignore_hidden_elements` to `false` before. When now calling  `wait_for_elements`, it will set the value back to `true` after the method call when `include_hidden` is true. You'll have to store the previous value of `Capybara.ignore_hidden_elements` and set the previous value after it.

To be honest, I think we should close this pull request and change the `ignore_hidden_elements` right in https://github.com/fastlane/produce/pull/39.